gpflow/conditionals/util.py

```diff
@@ -7,7 +7,7 @@
 def base_conditional(Kmn: tf.Tensor,
                      Kmm: tf.Tensor,
                      Knn: tf.Tensor,
-                     function: tf.Tensor,
+                     f: tf.Tensor,
```

Comment:
I agree that `function` name might not be appropriate. However, `f` is clearly not self-explanatory and depends a lot on the context. Moreover, we don't follow pep8 in naming, but still, we need to stick to those rules as far as it is possible.